ui/js/controllers/tcjobactions.js

```diff
@@ -1,4 +1,5 @@
 "use strict";
+var jsonSchemaDefaults = require('json-schema-defaults');
```

Comment:
I couldn't find any require's out side of the entrypoints -- is what you've suggested the recommended way ot do a require?